src/main/java/JobTracker.java

```diff
@@ -1,23 +1,40 @@
 import java.util.List;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
 
-public class JobTracker implements Runnable {
-    public void run() {
-     List<Job> activeJobs=Measurement.getJobs();
-     while(true){
-         //TODO synchronize appropriately as well as how often does the thread check
+public class JobTracker {
+    private final static ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
+
+    public static void startJobTracker(){
+        scheduler.scheduleAtFixedRate(trackJobs(),10,15, TimeUnit.MINUTES);
```

Comment:
The values there are they going to be in the config file to allow reconfigurability?

Reply:
Yeah will add those on a config